### ISSUE TITLE
Fails to build on Windows

### ISSUE THREAD

**Issue by MolotovCherry** [dignifiedquire/async-tar#48]
This fails to build on windows since [`async_std::os::windows::fs::symlink_file`](https://docs.rs/async-std/latest/async_std/os/windows/fs/fn.symlink_file.html) requires the `unstable` feature, but this feature is not enabled.

```rs
error[E0433]: failed to resolve: could not find `fs` in `windows`
   --> C:\Users\user\.cargo\registry\src\index.crates.io-6f17d22bba15001f\async-tar-0.4.2\src\entry.rs:588:41
    |
588 |                 async_std::os::windows::fs::symlink_file(src, dst).await
    |                                         ^^ could not find `fs` in `windows`
    |
note: found an item that was configured out
   --> C:\Users\user\.cargo\registry\src\index.crates.io-6f17d22bba15001f\async-std-1.12.0\src\os\windows\mod.rs:9:13
    |
9   |     pub mod fs;
    |
```

Offending line:
https://github.com/dignifiedquire/async-tar/blob/748fee5c70ee3aae297ff5e85ab11371bbb52407/src/entry.rs#L588

cc https://github.com/zed-industries/zed/issues/12079

**Comment by dignifiedquire** [comment]
thanks for the report, can you PR a fix, adjusting the feature?

**Comment by MolotovCherry** [comment]
> thanks for the report, can you PR a fix, adjusting the feature? 

Yes I can. I'll get one up within a few hours.

**Comment by MolotovCherry** [comment]
> thanks for the report, can you PR a fix, adjusting the feature? 

Yes I can. I'll get one up as soon as possible.

In the meantime, would you prefer this feature affect all targets, or only the windows target? Probably doesn't matter that much, but just checking.

Edit: PR is up now
Edit 2: Looks like there was already a PR up for this! #35

**Comment by dignifiedquire** [comment]
did some more cleanup and published `0.5.0` which should fix this
